static/css/base.css

```diff
@@ -0,0 +1,649 @@
+body {
+  top: 0;
+  left: 0;
+  color: rgba(51,51,51,1);
+}
+
+h1 {
+  font-family: "Roboto Slab", serif;
+  font-size: 1.75rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h2 {
+  font-family: "Roboto Slab", serif;
+  font-size: 1.75rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h3 {
+  font-family: "Roboto Slab", serif;
+  font-size: 1.5rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h4 {
+  font-family: "Open Sans", sans-serif;
+  font-size: 1.25rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h5 {
+  font-family: "Open Sans", sans-serif;
+  font-size: 1rem;
+  line-height: 125%;
+  font-weight: normal;
+}
+
+h6 {
+  font-family: "Open Sans", sans-serif;
+  font-size: 0.875rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+.header-text {
+  font-weight: normal;
+  color: rgba(250,250,250,1);
+  text-align: center;
+
+}
+
+.p-latestblog {
+  font-size: .875rem;
+}
+
+.font-weight-bolder {
+  font-weight: 600;
+}
+
+.offwhite-text {
+  color: rgba(250,250,250,1); /* #FAFAFA */
```

Comment:
Rather than having some comments indicating the hex counterparts for a particular `rgb()` / `rgba()` declaration, which will have to be diligently updated, I would prefer to see symbolic names indicating why those color values exist.

I don't know what our browser support strategy is (perhaps @rstorey can help?) but while the old-school approach would just be a text comment (“primary text color”, “highlighted element title color”, etc.), a more recent approach would be to use something like SCSS with a preprocessor (which I would support), and since it's 2018 we could go straight to using native CSS variables if we don't support IE11:

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables
https://caniuse.com/#feat=css-variables

(this could also be preprocessed or polyfilled)

Reply:
I think we'll have to support whatever browsers the public loc.gov website supports. I'm not sure what their strategy is but it would be easiest to stay in sync with that.

Reply:
Also, I'd like to extract this as a separate issue so it can be addressed throughout the CSS code and not hold up the pull request.